src/main/java/cms/gov/madie/measure/services/MeasureTransferService.java

```diff
@@ -30,9 +30,14 @@ public Measure overwriteExistingMeasure(
       draftdMeasures.sort(Comparator.comparing(Measure::getLastModifiedAt));
       Measure mostRecentMeasure = draftdMeasures.get(draftdMeasures.size() - 1);
       transferredMeasure.setId(mostRecentMeasure.getId());
+      List<TestCase> originalTestCases = mostRecentMeasure.getTestCases();
       if (GroupPopulationUtil.areGroupsAndPopulationsMatching(
           mostRecentMeasure.getGroups(), transferredMeasure.getGroups())) {
-        List<TestCase> originalTestCases = mostRecentMeasure.getTestCases();
+        transferredMeasure.setGroups(mostRecentMeasure.getGroups());
+        log.info(
+            "Overwrite meausre id {} with the testcases from original measure",
```

Comment:
I know you didn't write it, but could you fix the spelling of measure here?

Reply:
Done